apps/wallet/src/ui/app/pages/home/transfer-coin/FormOverlayStepper.tsx

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { ArrowRight16, ArrowLeft16 } from '@mysten/icons';
+import { Formik, type FormikConfig, type FormikValues } from 'formik';
+import { useCallback, useState, type ReactElement } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { Button } from '_app/shared/ButtonUI';
+import BottomMenuLayout, {
+    Content,
+    Menu,
+} from '_app/shared/bottom-menu-layout';
+import { SuiIcons } from '_components/icon';
+import Loading from '_components/loading';
+import Overlay from '_components/overlay';
+
+export interface FormStepProps
+    extends Pick<FormikConfig<FormikValues>, 'children' | 'validationSchema'> {
+    label: string;
+    loading?: boolean;
+}
+
+export function FormStep({ children }: FormStepProps) {
+    return <>{children}</>;
+}
+
+export function FormOverlayStepper({
+    children,
+    ...props
+}: FormikConfig<FormikValues>) {
+    const [showModal, setShowModal] = useState(true);
+    const navigate = useNavigate();
+    const closeSendToken = useCallback(() => {
+        navigate('/');
+    }, [navigate]);
+    const childrenArray = children as ReactElement<FormStepProps>[];
```

Comment:
Don't use `children` if your intent is to use it like an array, use another prop instead.

Reply:
I deleted this component